src/lib/main.js

```diff
@@ -44,29 +47,46 @@ const config = function (options = {}) {
     for (const optionPath of optionPaths) {
       // if DOTENV_KEY is set then assume we are checking envVaultFile
       if (DOTENV_KEY) {
-        envs.push({ type: 'envVaultFile', value: path.join(path.dirname(optionPath), '.env.vault') })
+        envs.push({
+          type: 'envVaultFile',
+          value: path.join(path.dirname(optionPath), '.env.vault')
+        })
       } else {
         envs.push({ type: 'envFile', value: optionPath })
       }
     }
 
-    const {
-      processedEnvs,
-      readableFilepaths,
-      uniqueInjectedKeys
-    } = new Run(envs, overload, DOTENV_KEY, processEnv).run()
+    const { processedEnvs, readableFilepaths, uniqueInjectedKeys } = new Run(
+      envs,
+      overload,
+      DOTENV_KEY,
+      processEnv
+    ).run()
 
     let lastError
+    /** @type {Record<string, string>} */
     const parsedAll = {}
 
     for (const processedEnv of processedEnvs) {
       if (processedEnv.type === 'envVaultFile') {
-        logger.verbose(`loading env from encrypted ${processedEnv.filepath} (${path.resolve(processedEnv.filepath)})`)
-        logger.debug(`decrypting encrypted env from ${processedEnv.filepath} (${path.resolve(processedEnv.filepath)})`)
+        logger.verbose(
+          `loading env from encrypted ${processedEnv.filepath} (${path.resolve(
+            processedEnv.filepath
+          )})`
+        )
+        logger.debug(
+          `decrypting encrypted env from ${
+            processedEnv.filepath
+          } (${path.resolve(processedEnv.filepath)})`
+        )
```

Comment:
The Prettier extension in my IDE also formatted the code because of some max characters per line rule.
I hope it's fine.

Reply:
yes, that's fine.